doc/sphinx/parameters/Boundary_20composition_20model.md

```diff
@@ -49,7 +49,7 @@ The following boundary composition models are available:
 
 Since the symbol $t$ indicating time may appear in the formulas for the prescribed composition, it is interpreted as having units seconds unless the global input parameter &ldquo;Use years in output instead of seconds&rdquo; is set, in which case we interpret the formula expressions as having units year.
 
-The format of these functions follows the syntax understood by the muparser library, see {ref}`sec:run-aspect:parameters-overview:muparser-format`.
+The format of these functions follows the syntax understood by the muparser library, see {ref}&lsquo;sec:run-aspect:parameters-overview:muparser-format&lsquo;.
```

Comment:
I think this is an incorrect replacement. The correct syntax for referencing section is using the backticks, but the script replaces them with the html equivalent. Can we modify the update script to not replace backticks? Or is this replacement necessary (I think in the past it was necessary because backtick is not an allowed latex character?).

Reply:
I am not sure why running the scripts ended up replacing these backticks only now as we had the replacement logic in there for a long time [1]. Note that I can not just delete this, because we are using single ticks `` `sin' `` in many places.


[1]: https://github.com/gassmoeller/aspect/blob/704ed414a7a2244322407ff20f56126858c47ab6/contrib/utilities/jsontomarkdown.py#L82